web/themes/custom/pineapple/components/_hero-image.twig

```diff
@@ -1,10 +1,13 @@
 {% set content_url = content_url|default('') %}
-<div class="hero-container breakout">
+{% set hero_attributes = attributes|default(create_attribute()).addClass([
```

Comment:
Update attributes to be `hero_attributes` also do this for line 6

Reply:
Done.